data/src/main/java/com/architect/coders/mu8/data/comics/ComicRepositoryImpl.kt

```diff
@@ -0,0 +1,26 @@
+package com.architect.coders.mu8.data.comics
+
+import com.architect.coders.mu8.data.service.MarvelServiceManager
+import com.architect.coders.mu8.data.utils.MARVEL_PUBLIC_KEY
+import com.architect.coders.mu8.data.utils.TIME_STAMP
+import com.architect.codes.mu8.callback.ComicRepository
+import com.architect.codes.mu8.comics.Comic
+import com.architect.codes.mu8.common.Scope
+
+class ComicRepositoryImpl(private val mapper: ComicsMapper) : ComicRepository,
+    Scope by Scope.Implementation() {
```

Comment:
We don't need this scope anymore on the repository since the ViewModel is the one that declares the Coroutine.